src/FileOnQ.Imaging.Heif/LibEncoder/LibEncoder_native.cs

```diff
@@ -5,7 +5,7 @@ namespace FileOnQ.Imaging.Heif
 {
 	partial class LibEncoder
 	{
-		private unsafe class x64
+		private unsafe class native
```

Comment:
I used lowercase for x64 and x86 because that is the convention for architecture. Since we are not using a special word or key anymore let's rename this to `Native`
```suggestion
		private unsafe class Native
```

You will need to update all references as well